openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/UserRepository.java

```diff
@@ -1054,6 +1059,12 @@ protected void postDelete(User entity, boolean hardDelete) {
     }
     // Remove suggestions
     daoCollection.suggestionDAO().deleteByCreatedBy(entity.getId());
+    CompletableFuture.runAsync(() -> updateIncidentAssignee(entity))
```

Comment:
I beleive there is common jobScheduler, shall we use that? check the importAsync or exportAsnc